pkg/artifactory/resource/repository/remote/remote.go

```diff
@@ -591,6 +600,17 @@ func verifyExternalDependenciesDockerAndHelm(_ context.Context, diff *schema.Res
 	return nil
 }
 
+func verifyDisableProxy(_ context.Context, diff *schema.ResourceDiff, _ interface{}) error {
+	disableProxy := diff.Get("disable_proxy").(bool)
```

Comment:
Do we need to use `GetOk()` instead here in case the attribute is not defined?

Reply:
This attribute has a default value, so it will be always defined.